### ISSUE TITLE
Release a version compatible with Django 1.9

### ISSUE THREAD

**Issue by charettes** [akaariai/django-reverse-unique#15]
The only released version is not compatible with Django 1.9

**Comment by charettes** [comment]
@holvianssi do you still have access to this repository or to the pypi credentials atleast?

I'd like to maintain it if you don't have time anymore.

**Comment by holvianssi** [comment]
Yeah, I think I can't maintain this, my schedule is way too unpredictable for reliable maintainership. I should have the pypi credentials somewhere, I'll check today at home.

**Comment by charettes** [comment]
Could you grant me _write_ rights to the repo so I can maintain it?

If you found the Pypi credentials you should be able to give me _Maintainer_ rights from there
https://pypi.python.org/pypi?:action=role_form&package_name=django-reverse-unique

**Comment by akaariai** [comment]
I invited you as a collaborator, not sure what else I need to do here.

I haven't yet found my pypi credentials yet... I recall storing them somewhere when I migrated from my old laptop. So, I'll keep searching.

**Comment by charettes** [comment]
Alright I'll push updates and adjust tests for Django up to 1.11. Keep me updated with the Pypi credentials.

**Comment by holvianssi** [comment]
@charettes you might want to check the work in https://github.com/django/django/pull/8238 and https://github.com/django/django/pull/7560 - the .filtered_relation() feature should be a nice replacement for this lib.

**Comment by charettes** [comment]
@holvianssi yeah I'm following it with a lot of interest! Thanks for the heads up.

**Comment by charettes** [comment]
@holvianssi did you retrieve your pypi credentials by any chance? I'm planning to use `filtered_relation()` but in the mean time I have to port an application to 1.8 to 1.11 (for LTS). If it's not the case I'll just publish a `django-reverse-unique-PREFIX` package to deal with this issue in the mean time. Thanks!

**Comment by charettes** [comment]
FWIW I've uploaded a temporary version compatible with up to Django 1.11 on pypi.
